tests/testthat/test_XcmsExperiment.R

```diff
@@ -633,7 +622,7 @@ test_that(".xmse_filter_peaks_intensities works", {
     res <- .xmse_filter_peaks_intensities(xmse, nValues = 1, threshold = 50000)
     expect_equal(res, unname(chromPeaks(xmse)[, "maxo"] >= 50000))
 
-    res <- .xmse_filter_peaks_intensities(xmse, nValues = 1, , msLeve = 2L)
+    res <- .xmse_filter_peaks_intensities(xmse, nValues = 1, , msLevel = 2L)
```

Comment:
Is the `, ,` on purpose ?

Reply:
It is not my code, i think there was some sort of error when copy/pasting. @jorainer Is it fine that I remove the extra comma ?

Reply:
hm, thanks for the catch @sneumann . Yes @philouail , please remove the extra comma.